Thorium-Sim/thorium-nova
# Solar system editor

## Description

Adds UI for creating and configuring stars and planets in the star map.

## Related Issue








Closes #188
Closes #189 
Closes #190 
Closes #191 
Closes #192 
Closes #194 
Closes #195 

## How do you know the changes work correctly?






Manual Testing:
Open the star map editor
Create a new solar system
Double click to enter the solar system
Create a new star
Create a new planet
Click to select the star or planet
Update the properties and see them persist.

## Screenshots (if appropriate):

## Types of changes



      change)

## Checklist:
